src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -56,6 +56,24 @@
                 "JobHeartbeatTimeoutThreshold": "00:10:00",
                 "JobPollingFrequency": "00:00:10",
                 "QueryDelayIntervalInMilliseconds": 500
+            },
+            "DataConvert": { 
+                "Enabled": true,
+                "ContainerRegistries": [
+                    {
+                        "ContainerRegistryServer": "",
+                        "ContainerRegistryUsername": "",
+                        "ContainerRegistryPassword": ""
+                    }
+                ],
+                "TemplateSetOptions": {
+                    "ShortCacheTimeSpan": "00:30:00",
+                    "LongCacheTimeSpan": "01:00:00",
+                    "ImageSizeLimitMegabytes": "10",
+                    "CacheMemoryLimitMegabytes": "60"
+                },
+                "ContainerRegistryTokenExpiration": "00:30:00",
+                "ProcessTimeoutThreshold": "00:01:00"
```

Comment:
>01 [](start = 47, length = 2)

nit: 
Just want to note what default value in code is 30 seconds
```
/// <summary>
/// Determines timeout for convert execution to terminate long running templates.
/// </summary>
public TimeSpan ProcessTimeoutThreshold { get; set; } = TimeSpan.FromSeconds(30);
```
not sure is it intended to have different config, or just out of sync.

Reply:
It's out of sync. Have changed to 30 seconds in the appsettings. Thanks